static/js/app.ts

```diff
@@ -184,7 +185,11 @@ export function initializeCodePage(options: InitializeCodePageOptions) {
   // *** EDITOR SETUP ***  
   const $editor = $('#editor');
   if ($editor.length) {
-    theGlobalEditor = editorCreator.initializeMainEditor($editor);
+    const dir = $("body").attr("dir");
+    theGlobalEditor = editorCreator.initializeWritableEditor($editor, EditorType.MAIN, dir);
+    attachMainEditorEvents();
+    error.setEditor(theGlobalEditor);
+    window.Range = ace.require('ace/range').Range // get reference to ace/range
```

Comment:
The weird part is that this is not being referenced anywhere, so I thought this might be a property of the `window` object. If that's not  the case maybe we can remove this.

Reply:
`window` is the object that represents the global scope.

So if you do:

```ts
window.banana = 'banana';
```
At any point later you can do

```ts
console.log(banana);
```
And it will work

Reply:
I understand. I removed the reference to windows.Range since it wasn't really used anywhere